test/extensions/filters/http/connect_grpc_bridge/connect_grpc_bridge_integration_test.cc

```diff
@@ -0,0 +1,73 @@
+#include "test/integration/http_integration.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace {
+
+class ConnectIntegrationTest : public testing::TestWithParam<Network::Address::IpVersion>,
```

Comment:
Can you have an integration test for the actual end to end flow for both unary and streaming?

In general can you also check the code coverage report and make sure you are reasonably covering all of the code?

Reply:
Done.

I did try to ensure the important cases are covered, and we have >96% on everything. We can aim for 100% if you would prefer though. I think the main tricky case is trying to find a way to get WKT JSON encoding to fail.